### PR TITLE
Quick fix for test_bootstrap setup method

### PR DESCRIPTION
**Pull request by oshtaier**
I felt that can cause an issue when reviewed PR, but was not sure for 100%

Fixing:
```
Error Message:
collection failure

Stacktrace:

tests/foreman/cli/test_bootstrap_script.py:7: in <module>
    class BootstrapScriptTestCase(CLITestCase):
tests/foreman/cli/test_bootstrap_script.py:11: in BootstrapScriptTestCase
    @classmethod
robottelo/decorators.py:136: in wrapper
    return unittest2.skip(reason)(pytest.mark.stubbed(func))
../../shiningpanda/jobs/ad95a8e7/virtualenvs/d41d8cd9/lib/python2.7/site-packages/unittest2/case.py:107: in decorator
    @wraps(test_item)
/usr/lib64/python2.7/functools.py:33: in update_wrapper
    setattr(wrapper, attr, getattr(wrapped, attr))
E   AttributeError: MarkDecorator instance has no attribute '__name__'
```

### CODE REVIEW

File: tests/foreman/cli/test_bootstrap_script.py [hunk 1]
```diff
@@ -5,9 +5,8 @@
 
 
 class BootstrapScriptTestCase(CLITestCase):
-    """Test class for bootstrap scipt."""
+    """Test class for bootstrap script."""
 
-    @stubbed()
     @classmethod
     def setUpClass(cls):
         """ create VM for testing """
```
Comment (abalakh) [inline tests/foreman/cli/test_bootstrap_script.py]: Ok, if it's unstubbed - it's also required to call `super()` here. And `# TODO: ` comment pointing that `setUpClass()` isn't actually fully implemented and needs to be updated would be useful too.

**Comment by abalakh** [comment]
ACK pending suggestion

File: tests/foreman/cli/test_bootstrap_script.py (hunk 1 above)
Comment (oshtaier) [inline tests/foreman/cli/test_bootstrap_script.py]: As for me ```""" create VM for testing """``` is informative enough. Of course, it isn't actually fully implemented as it is not implemented at all as whole module does. =)

**Comment by oshtaier** [comment]
Addressed comments

**Comment by jyejare** [comment]
ACK

**Pull request merged**